From ef3a197ba6981b853ce6fb8e433bbfa7a4e7d6a4 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Wed, 28 Feb 2018 10:01:40 +0800
Subject: [PATCH] power: rk817: fix compile warning

drivers/power/rk817_battery.c:2129:4: warning: this if
clause does not guard... [-Wmisleading-indentation]

drivers/power/rk817_battery.c:2920
rk817_bat_pm_resume() error: uninitialized symbol 'time_step'.

Change-Id: I9af73c85192279cc0546a5844da65542806279d5
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 drivers/power/rk817_battery.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/power/rk817_battery.c b/drivers/power/rk817_battery.c
index ca54c98c2836..b4a8d06cb8ad 100644
--- a/drivers/power/rk817_battery.c
+++ b/drivers/power/rk817_battery.c
@@ -2129,7 +2129,7 @@ static void rk817_bat_smooth_algorithm(struct rk817_battery_device *battery)
 			if (battery->smooth_soc < battery->dsoc)
 				return;
 
-				battery->dsoc = battery->smooth_soc;
+			battery->dsoc = battery->smooth_soc;
 			if (battery->dsoc <= 0)
 				battery->dsoc = 0;
 		}
@@ -2917,8 +2917,8 @@ static int rk817_bat_pm_resume(struct device *dev)
 	    interval_sec);
 
 	/* sleep: enough time and discharge */
-	if ((battery->sleep_dischrg_sec > time_step) &&
-	    (!battery->sleep_chrg_online)) {
+	if ((!battery->sleep_chrg_online) &&
+	    (battery->sleep_dischrg_sec > time_step)) {
 		if (rk817_bat_sleep_dischrg(battery))
 			battery->sleep_dischrg_sec = 0;
 	}
-- 
2.35.3

